components/keepassx/Makefile

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
Please, move it to appropriate category (desktop ? sysutil? http://wiki.openindiana.org/oi/Categories+layout ) and add REQUIRED_PACKAGES.